src/displayBackend/wayland/Display.cpp

```diff
@@ -428,24 +433,22 @@ void Display::registryHandler(wl_registry *registry, uint32_t id,
 	{
 		mCompositor.reset(new Compositor(mWlDisplay, registry, id, version));
 	}
-
-	if (interface == "wl_shell")
+	else if (interface == "wl_shell")
```

Comment:
This and similar changes bellow are not described in the commit message and probably should not belong to this particular patch.

Reply:
do you propose to have separate patch ?because , from my understanding, this approach is more clear and correct